l10n-pocket/en/pocket/platforms.ftl

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### iOS URL: https://www-dev-pocket.allizom.org/ios/
+### iOS URL: https://www-dev-pocket.allizom.org/android/
+### iOS URL: https://www-dev-pocket.allizom.org/safari/
+### iOS URL: https://www-dev-pocket.allizom.org/chrome/
+### iOS URL: https://www-dev-pocket.allizom.org/opera/
+### iOS URL: https://www-dev-pocket.allizom.org/edge/
+### iOS URL: https://www-dev-pocket.allizom.org/welcome/
```

Comment:
```suggestion
### Android URL: https://www-dev-pocket.allizom.org/android/
### Safari URL: https://www-dev-pocket.allizom.org/safari/
### Chrome URL: https://www-dev-pocket.allizom.org/chrome/
### Opera URL: https://www-dev-pocket.allizom.org/opera/
### Edge URL: https://www-dev-pocket.allizom.org/edge/
### Welcome URL: https://www-dev-pocket.allizom.org/welcome/
```

Reply:
Also, I'm not sure that FTL files can/should have more than one file level comment. We might need to switch these to group level comments (using `##` instead of `###`), and place them before their respective string content throughout the file.

Reply:
It's treated as a multiline file level comment. Still, using group comment is the best option here.